ui/src/app/cohort-search/list-overview/list-overview.component.ts

```diff
@@ -57,7 +57,10 @@ export class ListOverviewComponent implements OnChanges, OnInit {
   }
 
   ngOnChanges(changes: SimpleChanges): void {
-    if (changes.update && this.hasActiveItems && !this.hasTemporalError) {
+    if (changes.update &&
```

Comment:
Could this logic be in a a function that has a more descriptive name so it's easier to understand the condition?

Reply:
Good idea, done.